.github/ISSUE_TEMPLATE/feature.md

```diff
@@ -0,0 +1,11 @@
+# Issue: Feature
+> Note: The quickest way to see a new feature would be to submit a pull request, which will be reviewed when possible.
+## What feature?
+<!--
+Please describe what functionality this feature pertains to.
+-->
+
+## Why include this feature?
```

Comment:
I don't think we need this section. I believe users will describe the "why" in the feature description. We shouldn't ask them why they "feel" this feature should be included. Instead we should be interested in clear benefits of this feature.